pkg/virt-launcher/virtwrap/statsconv/converter.go

```diff
@@ -217,6 +221,6 @@ func Convert_libvirt_DomainJobInfo_To_stats_DomainJobInfo(info *libvirt.DomainJo
 		DataRemainingSet: info.DataRemainingSet,
 		DataRemaining:    info.DataRemaining,
 		MemDirtyRateSet:  info.MemDirtyRateSet,
-		MemDirtyRate:     info.MemDirtyRate,
+		MemDirtyRate:     mibToBytes(info.MemDirtyRate),
```

Comment:
Are you sure `MemDirtyRate` is in mibs? Refering to the docs https://libvirt.org/html/libvirt-libvirt-domain.html#VIR_DOMAIN_JOB_MEMORY_DIRTY_RATE:

> **number of memory pages** dirtied by the guest per second

Reply:
Thanks for the question. This indicates that this information might deserve a comment in the code.

Reply:
Well, maybe we can determine a page size then convert it to bytes? not sure if that would benefit us in any way

Reply:
https://libvirt.org/html/libvirt-libvirt-domain.html#VIR_DOMAIN_JOB_MEMORY_PAGE_SIZE
> memory page size in bytes, as VIR_TYPED_PARAM_ULLONG. If present, this parameter can be used to convert other page based statistics, such as VIR_DOMAIN_JOB_MEMORY_DIRTY_RATE or VIR_DOMAIN_JOB_COMPRESSION_PAGES to bytes.

Reply:
thank you for the catch, I was looking at: https://libvirt.org/html/libvirt-libvirt-domain.html under "VIR_DOMAIN_STATS_DIRTYRATE: Return memory dirty rate information." where it returns it in Mibs/
Yes I think multiplying by pagesize should work.

Reply:
@vasiliy-ul thank you, does this look better? (Will test manually also)

Reply:
I think you need to do something like that:
```go
		MemDirtyRateSet:  info.MemDirtyRateSet && info.MemPageSizeSet,
		MemDirtyRate:     info.MemDirtyRate * info.MemPageSize,
```
Because if `MemPageSize` is not set by libvirt you will have garbage in `MemDirtyRate`

Reply:
i think we may need this in first: https://gitlab.com/borod108/libvirt-go-module/-/merge_requests/1